docs/reference/commandline/docker.md

```diff
@@ -8,6 +8,7 @@ The base command for the Docker CLI.
 | Name                          | Description                                                                   |
 |:------------------------------|:------------------------------------------------------------------------------|
 | [`attach`](attach.md)         | Attach local standard input, output, and error streams to a running container |
+| [`bake`](bake.md)             | Build from a file                                                             |
 | [`build`](build.md)           | Build an image from a Dockerfile                                              |
 | [`builder`](builder.md)       | Manage builds                                                                 |
```

Comment:
Probably something we need to improve when generating these, and have an option to specify a custom target for these (so that `bake` can link to `https://docs.docker.com/buildx/bake.md`, and `attach` can link to `docker container attach` directly).

ISTR we had this for flags, but not sure now if we had it for commands.